### PR TITLE
Development

### PR DESCRIPTION
**Pull request by KhanyiCodes**
chatbot

### CODE REVIEW

**Comment by alphineandre** [comment]
Fix the buttons for the chatbot. You got this!!!

File: script.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
+let chatVisible = false;
+
+function toggleChat() {
+    const chatContainer = document.querySelector('.chat-container');
+    chatVisible = !chatVisible;
+    chatContainer.style.display = chatVisible ? 'block' : 'none';
+}
```
Comment (alphineandre) [inline script.js:7]:
```suggestion
function toggleChat() {
    const chatContainer = document.querySelector('.chat-container');
    chatVisible = !chatVisible;
    chatContainer.style.display = chatVisible ? 'block' : 'none';
}
```
use arrow functions

**Review by alphineandre — CHANGES_REQUESTED** [review]
submit before the end of the day.

**Comment by KhanyiCodes** [comment]
noted.

File: script.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
+let chatVisible = false;
+
```
Comment (alphineandre) [inline script.js:2]: Add more space to this

**Review by alphineandre — COMMENTED** [review]
Add space

File: index.html [hunk 3]
```diff
@@ -327,13 +328,48 @@ <h2>Get in Touch</h2>
...
+                <div class="bot-message">
+                    Hello! Welcome to Rebaone's portfolio. What can I help you find? 
+                </div>
+                <div class="bot-message">
+                    <button class="option-button" onclick="sendMessage('Home')">Home</button>
+                    <button class="option-button" onclick="sendMessage('About')">About Me</button>
+                    <button class="option-button" onclick="sendMessage('Skills')">Skills</button>
+                    <button class="option-button" onclick="sendMessage('Contact')">Contact Me</button>
+                </div>
+            </div>
```
Comment (alphineandre) [inline index.html:358]: @bokangkhumalo Do you think she should change this code?

**Review by alphineandre — COMMENTED** [review]
Check

File: index.html (hunk 3 above)
Comment (bokangkhumalo) [reply index.html:358]: Yes, must NOT include the nav bar in the chatbot

File: script.js (hunk 1 above)
Comment (Tepanyega) [reply script.js:7]: and also avoid using "querySelector" on "id's"

**Pull request merged**